### PR TITLE
redirected!

### PR DESCRIPTION
**Pull request by RaeChen07**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -41,10 +41,17 @@ const routes = [{
 // React Router
 const router = createBrowserRouter(routes);
 
-function App() {
-  return (
-    <RouterProvider router={router} />
-  )
+class App extends Component {
```
Comment (charliepnnl) [inline src/App.tsx:44]: I believe this can be done by modifying `routes` instead.

**Review by charliepnnl — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (RaeChen07) [reply src/App.tsx:44]: let me look into that...

File: src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import "bootstrap/dist/css/bootstrap.css";
-import React from "react";
-import {createBrowserRouter, RouterProvider} from 'react-router-dom'
+import React, { Component } from "react";
```
Comment (charliepnnl) [inline src/App.tsx:2]: Do you still need to import `Component`?

**Review by charliepnnl — CHANGES_REQUESTED** [review]
Please merge main into this branch.

**Review by ch-holly — APPROVED** [review]

**Review by charliepnnl — APPROVED** [review]

**Pull request merged**